server/profileFormSubmission.js

```diff
@@ -0,0 +1,35 @@
+const { Pool } = require('pg');
+const { allowedNodeEnvironmentFlags } = require('process');
+const pool = new Pool({ 
+    user: process.env.user,
+    host: process.env.host,
+    database: process.env.database,
+    password: process.env.password,
+    port: process.env.port,
+});
+
+
+module.exports.profileInsertion = async (event, callback )=> {
+    const body = JSON.parse(event.body);
+    const text = 'INSERT INTO cleaner_profile(first_name, last_name, contact_num) VALUES($1, $2, $3) RETURNING *'
```

Comment:
Could we change the name from text to like query or smt more descriptive?

Reply:
I know I'm not reviewer but quick q - why are we returning everything "*" instead of just id. do we need everything?

Reply:
Will we be adding the insertion of the other form data in a future sprint? ie. references, attachments, etc.
Eventually we'll have to make sure we're storing all the data from the form

Reply:
@nicolasabousawan when I try to return just the profile_id, it doesn't give the actual value for some reason. It's stored in the "rows" data rather than  the individual column.